org-roam-graph.el

```diff
@@ -97,10 +97,17 @@ Example:
   :type 'number
   :group 'org-roam)
 
-(defcustom org-roam-graph-wrap-title t
-  "When this is non-nil, wrap the title instead of truncating it.
-The value of `org-roam-graph-max-title-length` is used for wrapping."
-  :type 'boolean
+(defcustom org-roam-graph-short-titles t
+  "When this is non-nil, truncate titles in graph nodes.
+The title will be trancated according to `org-roam-graph-max-title-length'.
+
+This can also be set to symbol 'wrap, in which case long titles
+will be wrapped with `org-roam-graph-max-title-length' as the
+maximum line width."
+  :type '(choice
+          (const :tag "Yes" t)
```

Comment:
let's deprecate `org-roam-graph-max-title-length`, and move that configuration into this `defcustom`. so the choices are:

1. A number. If number, then truncate to this length.
2. `'wrap`. If `wrap`, wrap it.
3. `nil`. If nil, return the title as is.

Reply:
The problem here is that `s-word-wrap` also requires a number. It of course can be hard-coded, which is I guess less desirable. Or should it be passed together with `'wrap` in a cons cell? Like hardcoded default if just `'wrap` is provided and customizable `('wrap . 70)` to set wrapping after a particular number of symbols?

Reply:
Ah I see, I stand corrected. Could you just change `'t` to `'truncate` then? It's a bit more explicit than a truth value.